custom-login/pom.xml

```diff
@@ -68,6 +68,11 @@
             <artifactId>spring-boot-devtools</artifactId>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>com.nimbusds</groupId>
+            <artifactId>nimbus-jose-jwt</artifactId>
+            <version>8.20</version>
```

Comment:
There is a newer version `9.0` available but that is not backward compatible. So, sticking to `8.20`.